docs/README.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Installation guide
+label: Installation guide
+subline:
+description:
+preview:
+priority: 1
+keywords:
+  - Machine-learning
+  - Federated-learning
+  - Open-Source
+---
+
+# Installation guide
+
+To deploy `fl-studio` it is expected to have the following requirements fulfilled:
+
+## TODO
+
+* [ ] Update the [Start the FL-studio's federated learning training sample](#start-the-fl-studios-federated-learning-training-sample) screenshots to use the actual sample pipeline.
+* [ ] Update documentation to use `fl-studio` instead of `katulu-fl`
+
+## Requirements
+
+* [Docker](https://docs.docker.com/engine/install/) locally installed
+* A Kubernetes cluster (< `v1.21.x`) running, e.g [EKS](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html), [GKE](https://cloud.google.com/kubernetes-engine/docs/deploy-app-cluster#create_cluster), [AKS](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough#create-aks-cluster) or on-prem:
+  > 📝 Kubernetes < `v1.21.x` is needed due to [`no matches for kind "CustomResourceDefinition error`](https://github.com/kubeflow/manifests/issues/2028)
+* [kubectl installed](https://kubernetes.io/docs/tasks/tools/#kubectl) and [configured](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) to use the kubernetes cluster.
+* [kustomize installed](https://kubectl.docs.kubernetes.io/installation/kustomize/).
+* [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation): Kind is used to emulate an edge kubernetes cluster where the fl-edge client will run.
+* [curl](https://curl.se/download.html).
+* [python3](https://www.python.org/downloads/): To fetch the latest release tag.
+
+## Deploy fl-studio
+
+> 📝 the following deployment is not meant for a full production environment, e.g ingress tls is not configured, default username and password are hardcoded, etc
+
+### 1. Download the latest release assets:
+
+```shell
+LATEST_RELEASE_URL=$(curl -s -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/katulu-io/fl/releases/latest)
+FL_SUITE_MANIFESTS_URL=$(echo $LATEST_RELEASE_URL | python -c 'import json,sys;print(json.load(sys.stdin)["assets"][0]["url"])')
```

Comment:
Let's use `python3` instead (as macOS doesn't have `python` binaries), or require `jq` and use it here.